TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift

```diff
@@ -5,7 +5,16 @@
 //  Created by Keith Gan on 31/3/24.
 //
 
+import Foundation
+
 class InvulnerabilityPowerUp: EventTransformation {
+    let DURATION = CGFloat(5)
+    let id: UUID
+
+    init() {
```

Comment:
is the `id` here the same as the `id` in `EventTransformation` ? Is it necessary to repeat. Also should we instead make it extend identifiable so we enforce the need to have id if we are using it

Reply:
- this is an `EventTransformation` so the `id` is the same
- added the conformance to `Identifiable` for `EventTransformation`